go/.htaccess

```diff
@@ -5,14 +5,27 @@ RedirectMatch "/go/why\/?$" "https://marc.durdin.net/2018/03/the-case-for-keyman
 # Developer 10.0 onward redirects for package guide
 RedirectMatch "/go/(([1-9][0-9])([.]?)([0-9]))/developer-help-(mobile|packages)(/)?" "https://help.keyman.com/developer/$1/guides/distribute/packages"
 
-# TODO: Download redirects for keyboard permalinks (these three rules need refresh)
+# Download redirects for keyboard permalinks (TODO: these three rules need refresh)
```

Comment:
What's the TODO about?

Reply:
I don't know what the TODO for the 3 rules involve. They were carried over from the web.config.

https://github.com/keymanapp/keyman.com/blob/ab3d0a4dfcb1915a9dafd10e2d98fff0a90b4d03/go/web.config#L21-L35

Reply:
Lost in the mists of time. Probably unimportant, so we won't worry about it at this time.